test/mjsonwp/validator-specs.js

```diff
@@ -17,16 +17,19 @@ describe('MJSONWP', () => {
         (() => {validators.setUrl('foo');}).should.throw(/url/i);
       });
       it('should succeed when given url starting with http', async () => {
-        (() => {validators.setUrl('http://appium.io');}).should.not.throw;
+        (() => {validators.setUrl('http://appium.io');}).should.not.throw();
```

Comment:
seems like merge issue %)

Reply:
Fixed it. The W3C branch was behind master. Rebased and did a force push.